internal/init/init_test.go

```diff
@@ -48,6 +48,8 @@ func TestInitCommand(t *testing.T) {
 		assert.Error(t, Run(fsys))
 	})
 
+	// TODO: throws error on failure to find git root
+
```

Comment:
It should do nothing on failure to find git root, e.g. if the user isn't using git